### PR TITLE
Publish a tarball of Solana release binaries

### PR DESCRIPTION
**Pull request by pgarg66**

### CODE REVIEW

File: ci/publish-solana-tar.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+cd "$(dirname "$0")/.."
+
+DRYRUN=
+if [[ -z $BUILDKITE_BRANCH ]] || ./ci/is-pr.sh; then
```
Comment (mvines) [inline ci/publish-solana-tar.sh]: we probably don't need this check.  This script isn't run for PRs ever and it'll fail at the end without the aws credentials loaded anyway

Comment (pgarg66) [reply ci/publish-solana-tar.sh]: BUILDKITE_BRANCH check is valid though, right?

Comment (mvines) [reply ci/publish-solana-tar.sh]: I'd drop that check too actually.   CHANNEL will be empty if BUILDKITE_BRANCH is missing and we'll still bail

Comment (pgarg66) [reply ci/publish-solana-tar.sh]: So, no DRYRUN either?

**Comment by pgarg66** [comment]
@mvines , I think it's worth keeping the BUILDKITE_BRANCH check, as it allows DRYRUN. Let me know your thoughts.

**Comment by pgarg66** [comment]
@mvines , I am merging it for now. If you feel strongly about removing BUILDKITE_BRANCH check, I can push another change

**Pull request merged**